src/application/node/application.ts

```diff
@@ -23,6 +24,7 @@ export interface BackendApplicationContribution {
 export class BackendApplication {
 
     private app: express.Application;
+    @inject(ILogger) private logger: ILogger;
```

Comment:
With this, we just start to mix ctor and field injections. I am fine with that, I just do not know if your change was intentional or not.